save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestExecutionController.kt

```diff
@@ -226,7 +226,9 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
      */
     @PostMapping(value = ["/internal/saveTestResult"])
     fun saveTestResult(@RequestBody testExecutionsDto: List<TestExecutionDto>) = try {
-        if (testExecutionService.saveTestResult(testExecutionsDto).isEmpty()) {
+        if (testExecutionsDto.isEmpty()) {
```

Comment:
But in which case they will sent empty results?

Reply:
originally, I found it after issue in logic, but we have ability to exclude tests on test suite level. so we can select test suite with disabled tests and they can be run on a single agent (theoretically 😄 )